packages/components/src/components/icon/icon-styles.ts

```diff
@@ -34,6 +34,7 @@ const filterLightNotificationError =
   'invert(25%) sepia(62%) saturate(2003%) hue-rotate(335deg) brightness(100%) contrast(97%)';
 const filterLightNotificationInfo =
   'invert(31%) sepia(32%) saturate(5493%) hue-rotate(216deg) brightness(90%) contrast(107%)';
+const filterLightDisabled = filterLightContrastMedium;
```

Comment:
Seems to be an useless alias. Why not reuse the variable in the object mapping right away?